.github/workflows/chromatic.yaml

```
@@ -28,7 +28,8 @@ jobs:
       - run: |
           echo "runChromatic=${runChromatic}" >> $GITHUB_ENV
         env:
-          runChromatic: ${{ github.event.review.state == 'approved' }}
+          # If action is running on master branch we would like to update baselines via autoAcceptChanges
+          runChromatic: ${{ github.event.review.state == 'approved' || github.ref == 'refs/heads/master' }}
```

Apparently we should be able to do something like this
```suggestion
          runChromatic: ${{ github.event.review.state == 'approved' || ${GITHUB_REF##*/} == 'master' }}
```

https://stackoverflow.com/questions/58033366/how-to-get-the-current-branch-within-github-actions

---

I have a slight preference for the longer form as it avoids bash magic. TIL though! Thanks for sharing.